src/01-Learning-JS/06-strings.md

```diff
@@ -203,9 +213,19 @@ Ok, last exercise! Complete the following steps to create a new array that only
 1. In a first codeblock, declare a new array called `hyphenatedWords`. Then, push only hyphenated words into it.
 2. In a second codeblock, output the new array to verify your work.
 
+```js
+let hyphenatedWords = []
+for (const word of vonnSplit){
+  if (word.includes("-")){
+    hyphenatedWords.push(word)
+  }
+}
+console.log(hyphenatedWords)
+```
+
 **Question**: What oddities do you notice about the outcome? Below, explain what you suggest is happenning, and what you would do to resolve the issue with isolating a better list of hyphenated words.
 
-ENTER_YOUR_RESPONSE_HERE
+Some of the hyphenated words include strings of \r\n\r\n followed by the next word in the short story. This is being included because there are no spaces between the hyphenated word and the rest of this string. To resolve this, I might remove the instances of \r\n... first then push the hyphenated words to a new array.
```

Comment:
In this case, I wanted to highlight what those other types of characters were, such as:

```
"\n" // 1 new line
"\r" // 1 return carriage

// Others to note:
"\s" // 1 whitespace
"\t" // 1 tab
```

If we wanted to remove them, we could use these char types in a replace method.